digitaltwin_client/samples/digitaltwin_sample_environmental_sensor/digitaltwin_sample_environmental_sensor.c

```diff
@@ -603,23 +605,16 @@ DIGITALTWIN_CLIENT_RESULT DigitalTwinSampleEnvironmentalSensor_SendTelemetryMess
     float currentTemperature = 20.0f + ((float)rand() / RAND_MAX) * 15.0f;
     float currentHumidity = 60.0f + ((float)rand() / RAND_MAX) * 20.0f;
 
-    char currentTemperatureMessage[128];
-    char currentHumidityMessage[128];
+    char currentMessage[128];
 
-    sprintf(currentTemperatureMessage, "%.3f", currentTemperature);
-    sprintf(currentHumidityMessage, "%.3f", currentHumidity);
+    sprintf(currentMessage, "{\"%s\":%.3f, \"%s\":%.3f}", 
```

Comment:
>sprintf(currentMessage, "{\"%s\":%.3f, \"%s\":%.3f}",  [](start = 4, length = 54)

This is legal JSON but it's not JSON that's friendly with how DTML wants it.  There needs to be something like 
{ "Environment" { "temperature":123, "humidity"23}} e.g. - I'm making the "Environment" bit up.

We should loop back with @TanmayBhagwat to understand what the post-telemetry explosion DTML for enviromental sensor will look like and code the correct "Environment" thingy in.